### PR TITLE
[Feature-1883][Plugin] Implement the gitee plugin

### PR DESCRIPTION
**Pull request by SnowMoon-Dev**
# Summary
Implement the gittee plugin,

### Does this close any open issues?
close [#1883](https://github.com/apache/incubator-devlake/issues/1883)

### Screenshots
image

### Other Information
Any other information that is important to this PR.

### CODE REVIEW

**Comment by warren830** [comment]
I tried to run all tasks locally, but I got the following error:
image

File: plugins/gitee/tasks/commit_extractor.go [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+	}
+
+	return extractor.Execute()
+}
+
+// ConvertCommit Convert the API response to our DB model instance
+func ConvertCommit(commit *GiteeApiCommitResponse) (*models.GiteeCommit, error) {
+	giteeCommit := &models.GiteeCommit{
+		Sha:            commit.Sha,
+		AuthorId:       commit.Author.Id,
```
Comment (warren830) [inline plugins/gitee/tasks/commit_extractor.go:117]: I tried to run all tasks locally, but I got the following error:
image
I think this is the root clause, commit.Author might be nil

File: plugins/gitee/tasks/pr_review_collector.go [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+var CollectApiPullRequestReviewsMeta = core.SubTaskMeta{
+	Name:             "collectApiPullRequestReviews",
+	EntryPoint:       CollectApiPullRequestReviews,
+	EnabledByDefault: true,
+	Description:      "Collect PullRequestReviews data from Gitee api",
+}
+
+func CollectApiPullRequestReviews(taskCtx core.SubTaskContext) error {
+	db := taskCtx.GetDb()
```
Comment (warren830) [inline plugins/gitee/tasks/pr_review_collector.go:43]: got `not found project` error when collecting pr_review
image

**Review by warren830 — CHANGES_REQUESTED** [review]
Great Job! Only a little minor bugs need to be fixed

File: plugins/gitee/tasks/pr_review_collector.go (hunk 2 above)
Comment (SnowMoon-Dev) [reply plugins/gitee/tasks/pr_review_collector.go:44]: ok, i will fixed it

File: plugins/gitee/tasks/commit_extractor.go (hunk 1 above)
Comment (SnowMoon-Dev) [reply plugins/gitee/tasks/commit_extractor.go]: ok, i will fixed it

**Review by warren830 — APPROVED** [review]
LGTM

**Pull request merged**